tests/istio.mk

```diff
@@ -180,22 +180,3 @@ test/minikube/auth/e2e_pilot_alpha1: istioctl
         --ns pilot-auth-system \
         -n pilot-test \
            ${TESTOPTS} | tee ${OUT_DIR}/tests/test-report-auth-pilot-v1.raw
-
```

Comment:
Why remove this target ? It is used in circle.

Reply:
There's two of them.  Any invocation of make complains:
<pre>
tests/istio.mk:186: warning: overriding recipe for target 'test/minikube/noauth/e2e_pilot_alpha1'
tests/istio.mk:128: warning: ignoring old recipe for target 'test/minikube/noauth/e2e_pilot_alpha1'
</pre>

Reply:
thanks for fixing this @mattdelco